requirements/requirements.in

```diff
@@ -116,3 +116,4 @@ WeasyPrint==0.42.3
 architect==0.5.6
 prometheus-client==0.7.1
 turn-python==0.0.1
+django-logentry-admin==1.0.6
```

Comment:
note that these these are already accessible on the django admin via the 'history' view of a user. Not saying this isn't also useful but just FYI.

Reply:
Oh I see..
this view just gives a little more information like action (addition/change) and you can see actions taken by a user so you can see all users created by a user, which can be handy so not removing this.